scripts/bench.ts

```diff
@@ -0,0 +1,22 @@
+import execa from "execa";
+import globby from "globby";
+import path from "path";
+
+async function main() {
+  const benchmarks = await globby("./src/packages/**/*.bench.ts", {
+    gitignore: true,
+  });
+  await run(benchmarks);
+}
+async function run(benchmarks: string[]) {
+  for (const location of benchmarks) {
+    await execa.command(`yarn ts-node ${location}`, {
```

Comment:
```suggestion
    await execa.command(`pnpm run ts-node ${location}`, {
```
Maybe better to use pnpm everywhere

Reply:
:fire: https://github.com/prisma/prisma/pull/5301/checks?check_run_id=1778572311#step:8:17